extensions/wrapper/wrapper/src/test/java/de/sovity/edc/ext/wrapper/api/ui/pages/policy/PolicyDefinitionApiServiceTest.java

```diff
@@ -81,39 +92,53 @@ void getPolicyList() {
     }
 
     @Test
-    void test_sorting(PolicyDefinitionService policyDefinitionService) {
+    void sortPoliciesFromNewestToOldest(DSLContext dsl) {
         // arrange
-        createPolicyDefinition(policyDefinitionService, "my-policy-def-2", 1628956802000L);
-        createPolicyDefinition(policyDefinitionService, "my-policy-def-0", 1628956800000L);
-        createPolicyDefinition(policyDefinitionService, "my-policy-def-1", 1628956801000L);
+        createPolicyDefinition("my-policy-def-0");
+        createPolicyDefinition("my-policy-def-1");
+        createPolicyDefinition("my-policy-def-2");
+
+        val dates = Map.of(
+            "my-policy-def-0", 1628956800000L,
+            "my-policy-def-1", 1628956801000L,
+            "my-policy-def-2", 1628956803000L
+        );
+
+        val p = Tables.EDC_POLICYDEFINITIONS;
+        dates.forEach((id, time) ->
+            dsl.update(p)
+                .set(p.CREATED_AT, time)
+                .where(p.POLICY_ID.eq(id))
+                .execute()
+        );
```

Comment:
this is great, but I think it should be in its own method. Many lines for a small api surface. Why force readers to parse this block when you could just have

```java
dates.forEach(id, time) -> this.setPolicyDefCreatedAt(id, time));
```